core/src/test/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotServiceSpec.scala

```diff
@@ -993,6 +1008,44 @@ class SnapshotServiceSpec extends AnyWordSpecLike with Matchers with MockitoSuga
       }
     }
 
+    "create a new snapshot reference when called with workspaceId" in withMinimalTestDatabase { _ =>
+      val mockSamDAO = defaultMockSamDao()
+      val mockWorkspaceManagerDAO = defaultMockWorkspaceManagerDao()
+      val mockDataRepoDAO = defaultDataRepoDao()
+      val workspace = minimalTestData.workspace
+
+      val snapshotService = SnapshotService.constructor(
+        slickDataSource,
+        mockSamDAO,
+        mockWorkspaceManagerDAO,
+        "fake-terra-data-repo-dev",
+        mockDataRepoDAO
+      )(testContext)
+
+      val snapshotUuid = UUID.randomUUID()
+      val snapRefName = DataReferenceName("refname")
+      val snapRefDescription = Option(DataReferenceDescriptionField("my reference description"))
+
+      // call createSnapshot on the service
+      Await.result(
+        snapshotService.createSnapshotByWorkspaceId(workspace.workspaceIdAsUUID.toString,
+                                                    NamedDataRepoSnapshot(snapRefName, snapRefDescription, snapshotUuid)
+        ),
+        Duration.Inf
+      )
+
+      // assert that the service called WSM's createDataRepoSnapshotReference
+      verify(mockWorkspaceManagerDAO).createDataRepoSnapshotReference(
+        ArgumentMatchers.eq(workspace.workspaceIdAsUUID),
```

Comment:
nit: Any chance you can import `eq` the same way `any` is imported, and eliminate the extra-verbose `ArgumentMatchers` prefix?